tuf/repository_lib.py

```diff
@@ -1500,6 +1543,16 @@ def generate_timestamp_metadata(snapshot_filename, version, expiration_date,
       An object which implements
       securesystemslib.storage.StorageBackendInterface.
 
+    use_length:
+      Used to decide if the length attribute should be used. From version 1.0.1
```

Comment:
Same as above, this implies it's only true since this spec version. Does including a version in this docstring help the reader?

Reply:
Yes, you are right. In tuf spec version `1.0.0` they were optional too.
My mistake.